index.html

```diff
@@ -918,6 +921,10 @@
 people need. For example, some user agents may help people present different
 [=identities=] to subdivisions of a single [=site=].
 
+In each of these [=contexts=] the UA should assume by default that the user
```

Comment:
I was uncertain if this text should be elevated to the principle text box because I hadn't spent enough time looking into what should and should not go within there. I put it below since the text right above this talks about defaults, but I'd be happy to see this text go within the principle text box if others also believe it warrants that level of attention.